fundamentals-3.md: add additional resource

AshkanArabim opened the pull request:
Added a link to a W3schools page about JavaScript function parameters that also explains how the 'arguments' object works.

<!-- Thank you for taking the time to contribute to The Odin Project. In order to get a pull request (PR) closed in a reasonable amount of time, you must include a baseline of information about the changes you are proposing. Please read this template in its entirety before filling it out to ensure that it is filled out correctly. -->

Complete the following REQUIRED checkboxes:
<!-- While editing this template, replace the whitespace between the square brackets with an 'x', e.g. [x] -->
-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`

Complete the following checkboxes ONLY IF they are applicable to your PR. You can complete them later if they are not currently applicable:
-   [x] I have previewed all lesson files included in this PR with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure the Markdown content is formatted correctly
-   [x] I have ensured all lesson files included in this PR follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

<hr>

**1. Because:**
<!--
If this PR closes an open issue, replace the XXXXX below with the issue number, e.g. Closes #2013. Or if the issue is in another TOP repo replace the #XXXXX with the URL of the issue, e.g. Closes https://github.com/TheOdinProject/curriculum/issues/XXXXX

Otherwise, provide a clear and concise reason for your pull request, e.g. what problem it solves or what benefit it provides. If this PR is related to, but does not close, another issue or PR, you can also link it as above without the 'Closes' keyword, e.g. "Related to #2013".
 -->
Closes #XXXXX


**2. This PR:**
<!--
A bullet point list of one or more items outlining what was done in this PR to solve the problem(s) or implement the feature/enhancement.
 -->


**3. Additional Information:**
<!-- Any additional information about the PR, such as a link to a Discord discussion, etc. -->

wise-king-sullyman reviewed (commented):
I think that this is a good topic to add an additional resource about, but I wonder if this lesson is an ideal place to add it since we don't cover objects until fundamentals five. I'll think on it and circle back 🙂

wise-king-sullyman reviewed (changes requested):
Per my previously discussed reasoning I think this would be a better additional resource in fundamentals 5, can you move it to that lesson?

AshkanArabim commented:
I removed the additional resource about arguments that I had added before. I will open another pull request to add it to the 'fundamentals 5' section.

wise-king-sullyman commented:
@AshkanArabim you can just add it to that lesson in this PR if you'd like, a new PR isn't needed.

AshkanArabim commented:
I made the requested changes.
I noticed that in fundamentals-5.md, unlike the others, the additional resources section was after the knowledge check section. I made the correction.

wise-king-sullyman reviewed (changes requested):
Can you actually move the additional resources back to below the knowledge check section? That's the layout that we're going for in the future per [our lesson template](https://github.com/TheOdinProject/curriculum/blob/main/templates/lesson-template.md).

After that this should be good to go! 🙂

wise-king-sullyman commented on foundations/javascript_basics/fundamentals-5.md:
```diff
@@ -50,4 +50,4 @@ This section contains questions for you to check your understanding of this less
 
-*   It looks like this lesson doesn't have any additional resources yet. Help us expand this section by contributing to our curriculum.
\ No newline at end of file
+*   Learn about the 'arguments' object on [this page](https://www.w3schools.com/js/js_function_parameters.asp) about function parameters.
```
```suggestion
*   Learn about the 'arguments' object on this [page about function parameters](https://www.w3schools.com/js/js_function_parameters.asp).
```

wise-king-sullyman reviewed (changes requested):
Actually I have one final tweak I'd like to see, just a slight update of the link text used to keep things more accessible.

You can read about the reasoning behind this suggestion in this great [article about link text from Harvard](https://accessibility.huit.harvard.edu/technique-writing-link-text).

wise-king-sullyman reviewed (approved):
Looks perfect, thanks!
